nemo/collections/nlp/data/language_modeling/megatron/gpt_sft_chat_dataset.py

```diff
@@ -96,6 +116,26 @@ def _add_speaker_and_signal(header, source, mask_role, gtype):
                 + sentence["value"]
                 + END_SIGNAL
             )
+        elif gtype == "VALUE_TO_TEXT":
+            sentence["value"] = (
+                BEGIN_SIGNAL
+                + role_token
+                + sentence_from
+                + END_NAME_SIGNAL
+                + (response_value_formater(sentence['label']) if 'label' in sentence else '')
```

Comment:
Can you leave a comment here indicating that loss is not calculated on `(response_value_formater(sentence['label']) if 'label' in sentence else '')` ? Don't think it's very straightforward here.

Reply:
it is not about loss calculation but to construct the conversation according to dialogue templates.

Reply:
ok think this was addressed elsewhere